### DISCUSSION TITLE
Unoplat cli basic components

### DISCUSSION THREAD

**Discussion by JayGhiya** [unoplat/.github#16] [Ideas]
MVP: 
- First Time Setup (Github Token Acquisition ,kubeconfig Acquisition, Enabling Github to be used as chart repository, Service Mesh,Performance Testing, Observability Installation)
- Application templates
- Data Processing templates
- Databases - (Message Brokers/Relational/NonRelational/Vector)
- Auth - (AuthZ, AuthN & Externalise Services - Here we will be abstracting consul, open policy agent and gateway services for the user)
- Observability - (Abstracting Dashboards, Alerting and Logging for Installed Services)

**Comment by JayGhiya** [comment]
please provide feedback @vipinshreyaskumar

**Reply by vipinshreyaskumar** [reply]
+1 on the MVP. feature set looks good to me.

**Reply by vipinshreyaskumar** [reply]
env variables needed for cli : **KUBECONFIG** , **GITHUB_TOKEN**
